css-brochure-form/styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+  font-family: "Brandon Grotesque", sans-serif;
+}
+
+body {
+  margin-top: 40px;
```

Comment:
You should not really apply margin to the body, you don't want to push the entire content of the page away from the viewport. The `body` _is_ the content of the viewport :joy:. You should use padding instead.

Reply:
Yes, that makes sense! Thanks Cody!!